src/components/help-text/help-text.component.tsx

```diff
@@ -38,7 +37,7 @@ export const HelpText = ({ title, helpTextAbstract, helpTextDescription, require
           <p
             className="help-text mb-0"
             dangerouslySetInnerHTML={{
```

Comment:
Is it possible to remove dangerouslySetInnerHTML entirely?

Reply:
helptekst er html

Reply:
oh. burde sjekke før pr! helptekst er faktisk markdown 😄